<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sgsApp</class>
 <widget class="QMainWindow" name="sgsApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1440</width>
    <height>900</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1440</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Student Grading System</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidgetSGS">
    <property name="geometry">
     <rect>
      <x>-3</x>
      <y>-3</y>
      <width>1451</width>
      <height>911</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="loginScreen">
     <property name="styleSheet">
      <string notr="true">background:url(:/images/background.jpg);
background-position:center;
background-repeat: no-repeat;</string>
     </property>
     <widget class="QFrame" name="loginBox">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>250</y>
        <width>291</width>
        <height>331</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid #8a1738;
border-radius:10px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLineEdit" name="usernameInput">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>90</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background:#fff;
color: #333;
border:0px; 
border-top-right-radius:5px;
border-bottom-right-radius:5px;
border-top-left-radius:0px;
border-bottom-left-radius:0px;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="passwordInput">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>140</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:#fff;
color: #333;
border:0px; 
border-top-right-radius:0px;
border-bottom-right-radius:0px;
border-top-left-radius:0px;
border-bottom-left-radius:0px;</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="signInButton">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>190</y>
         <width>231</width>
         <height>32</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:#fbaf01;
color: #ffffff;
border:0;
height:30px;
border-radius:5px;</string>
       </property>
       <property name="text">
        <string>SIGN IN</string>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/images/lock.png</normaloff>:/images/lock.png</iconset>
       </property>
      </widget>
      <widget class="QPushButton" name="signUpButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>280</y>
         <width>113</width>
         <height>32</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:0px; color:#fff;</string>
       </property>
       <property name="text">
        <string>SIGN UP</string>
       </property>
      </widget>
      <widget class="QPushButton" name="forgotPasswordButton">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>280</y>
         <width>113</width>
         <height>32</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:0px;
color:#fff;
text-decoration:underline;</string>
       </property>
       <property name="text">
        <string>Forgot Password?</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>30</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #fff; font-size:28px; border:0px;</string>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
      <widget class="QFrame" name="usernameIcon">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>90</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:#fff url(:/images/user.png);
border:0px;
border:0px;
border-top-left-radius:5px;
border-bottom-left-radius:5px;
border-top-right-radius:0px;
border-bottom-right-radius:0px; 
</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
      <widget class="QFrame" name="passwordIcon">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>140</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:#fff url(:/images/lock.png);
border:0px;
border-radius:0px;
border-top-left-radius:5px;
border-bottom-left-radius:5px;
border-top-right-radius:0px;
border-bottom-right-radius:0px;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
      <widget class="QLabel" name="loginErrorLabel">
       <property name="geometry">
        <rect>
         <x>49</x>
         <y>240</y>
         <width>211</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:0px;
color:#b50009;
text-align:center;
background: transparent;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>140</y>
         <width>21</width>
         <height>31</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>31</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background:#fff;
border:0px; 
border-top-right-radius:5px;
border-bottom-right-radius:5px;
border-top-left-radius:0px;
border-bottom-left-radius:0px;</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="footerLogo">
      <property name="geometry">
       <rect>
        <x>990</x>
        <y>180</y>
        <width>191</width>
        <height>61</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>191</width>
        <height>61</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>191</width>
        <height>61</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background:url(:/images/sideBarLogo.png);
background-position:center;
background-repeat: no-repeat;
border: 0px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>260</y>
        <width>462</width>
        <height>281</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>462</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background:url(:/images/logo.png);
border:0px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Dashboard">
     <property name="minimumSize">
      <size>
       <width>1440</width>
       <height>900</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1440</width>
       <height>900</height>
      </size>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>-20</x>
        <y>-10</y>
        <width>2</width>
        <height>2</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4"/>
     </widget>
     <widget class="QFrame" name="Sidebar">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>260</width>
        <height>900</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>260</width>
        <height>900</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>260</width>
        <height>900</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#3a444c;
border: 0;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="frame">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>730</y>
         <width>191</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:url(:/images/sideBarLogo.png);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>30</y>
         <width>181</width>
         <height>401</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QGroupBox" name="mainMenu_groupBox">
          <property name="styleSheet">
           <string notr="true">color:#70808c;</string>
          </property>
          <property name="title">
           <string>Main Menu</string>
          </property>
          <widget class="QWidget" name="layoutWidget_3">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>30</y>
             <width>128</width>
             <height>91</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QFormLayout" name="formLayout">
              <property name="labelAlignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
              <property name="formAlignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
              <item row="0" column="0">
               <widget class="QPushButton" name="dashboard_pushButton">
                <property name="layoutDirection">
                 <enum>Qt::LeftToRight</enum>
                </property>
                <property name="styleSheet">
                 <string notr="true">color:#fff;</string>
                </property>
                <property name="text">
                 <string>Dashboard</string>
                </property>
                <property name="icon">
                 <iconset resource="images.qrc">
                  <normaloff>:/images/user.png</normaloff>:/images/user.png</iconset>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QFormLayout" name="formLayout_2">
              <property name="labelAlignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
              <property name="formAlignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
              <item row="0" column="0">
               <widget class="QPushButton" name="AddProgramSequence_pushButton">
                <property name="layoutDirection">
                 <enum>Qt::LeftToRight</enum>
                </property>
                <property name="styleSheet">
                 <string notr="true">color:#fff;</string>
                </property>
                <property name="text">
                 <string>Program Sequence</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="accounts_groupBox">
          <property name="styleSheet">
           <string notr="true">color:#70808c;</string>
          </property>
          <property name="title">
           <string>Accounts</string>
          </property>
          <widget class="QWidget" name="layoutWidget_4">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>40</y>
             <width>117</width>
             <height>48</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <layout class="QFormLayout" name="formLayout_5">
              <property name="labelAlignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
              <property name="formAlignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
              <item row="0" column="0">
               <widget class="QPushButton" name="dashboard_pushButton_2">
                <property name="layoutDirection">
                 <enum>Qt::LeftToRight</enum>
                </property>
                <property name="styleSheet">
                 <string notr="true">color:#fff;</string>
                </property>
                <property name="text">
                 <string>Student Account</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QFormLayout" name="formLayout_4">
              <property name="labelAlignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
              <property name="formAlignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
              <item row="0" column="0">
               <widget class="QPushButton" name="AddProgramSequence_pushButton_2">
                <property name="layoutDirection">
                 <enum>Qt::LeftToRight</enum>
                </property>
                <property name="styleSheet">
                 <string notr="true">color:#fff;</string>
                </property>
                <property name="text">
                 <string>Lecturer Account</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="settings_groupBox">
          <property name="styleSheet">
           <string notr="true">color:#70808c;</string>
          </property>
          <property name="title">
           <string>Settings</string>
          </property>
          <widget class="QWidget" name="layoutWidget_5">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>42</y>
             <width>128</width>
             <height>76</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <layout class="QFormLayout" name="formLayout_8">
              <property name="labelAlignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
              <property name="formAlignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
              <item row="0" column="0">
               <widget class="QPushButton" name="resetPasswd_pushButton">
                <property name="layoutDirection">
                 <enum>Qt::LeftToRight</enum>
                </property>
                <property name="styleSheet">
                 <string notr="true">color:#fff;</string>
                </property>
                <property name="text">
                 <string>Reset Password</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QFormLayout" name="formLayout_9">
              <property name="labelAlignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
              <property name="formAlignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
              <item row="0" column="0">
               <widget class="QPushButton" name="addProgamSeqSettings_pushButton">
                <property name="layoutDirection">
                 <enum>Qt::LeftToRight</enum>
                </property>
                <property name="styleSheet">
                 <string notr="true">color:#fff;</string>
                </property>
                <property name="text">
                 <string>Program Sequence</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QFormLayout" name="formLayout_11">
              <property name="labelAlignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
              <property name="formAlignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
              <item row="0" column="0">
               <widget class="QPushButton" name="addProgamSeqSettings_pushButton_2">
                <property name="layoutDirection">
                 <enum>Qt::LeftToRight</enum>
                </property>
                <property name="styleSheet">
                 <string notr="true">color:#fff;</string>
                </property>
                <property name="text">
                 <string>Suspend User</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QFrame" name="headBar">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1440</width>
        <height>50</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>1440</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>260</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#3c8dbc;
border:0;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="lecturerDashboard">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>439</x>
        <y>230</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Lecturer Dashboard</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="studentDashboard">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>250</y>
        <width>211</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Student Dashboard</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
